### DISCUSSION TITLE
Mesh Size ofe Rigid Bodies and Time step

### DISCUSSION THREAD

**Discussion by Hyp5891** [OpenRadioss/OpenRadioss#2403] [Q&A]
Hi,

my question is: does the mesh size of a parts influence the time step / run time when these parts are defined as rigid (ls-dyna MAT20)?

My idea was to rigidize parts that I am not especially interested in but keep the system level, mass contribution and contact interaction. Imagine for example a bicylce assembly running over a bump. Somehow close to what can be done using MBD simulation keeping things rigid you are not interessted in but to make other bodies "flexible".

I recently tried to speed up a bigger model by making many parts rigid but it seemed to not have a big influence.

Can somebody give some insights?

Thanks in advance!

**Comment by MarianBulla** [comment]
Hi @Hyp5891,

yes, this is true. Parts, which are completely rigid, do not influence the timestep (the timestep is infinite for those parts). These elements are also not used for calculation of strain and stress. So they cost almost nothing.

And yes, we have also this bicycle model, which is rigid untild hitting a bump. Then it is switched to "deformable" by switching the rigid-body-OFF.
So, it is also doable in OpenRadioss.

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by Hyp5891** [reply]
Hi @MarianBulla,

thanks for your fast and helpful answer! That's really cool to hear that it can be done like that. Also the idea of switching to "deformable" can be benefical I think. I will keep it in mind and experiment a little bit further with this rigid body approach.

Best regards,
Christian

**Reply by MarianBulla** [reply]
Hi Christian,

this is great to hear. 

Please let me know, if you need an example for this.

Thanks and best regards,
Marian
